src/components/simulation/ElectricityConsumptionManual.vue

```diff
@@ -22,6 +24,15 @@ function inputIsCompleted() {
     return true
   }
 }
+
+onBeforeMount(() => {
+  if (enedisStore.isEnedisRedirection) {
+    const annual_consumption = Math.round(
+      consumptionAndProductionStore.annualConsumption
+    )
+    consumption.value = annual_consumption.toString()
```

Comment:
Any reason why it is a string? *just curious

Reply:
There is no specific reason it's true, we would have put it in number